replications/service.go

```diff
@@ -424,3 +426,40 @@ func (s service) populateRemoteHTTPConfig(ctx context.Context, id platform.ID, t
 
 	return nil
 }
+
+func (s service) Open(ctx context.Context) error {
+	var trackedReplications influxdb.Replications
+
+	// Get replications from sqlite
+	q := sq.Select(
+		"id", "max_queue_size_bytes").
+		From("replications")
+
+	query, args, err := q.ToSql()
+	if err != nil {
+		return err
+	}
+
+	if err := s.store.DB.SelectContext(ctx, &trackedReplications.Replications, query, args...); err != nil {
+		return err
+	}
+
+	trackedReplicationsMap := make(map[platform.ID]int64)
+	for _, r := range trackedReplications.Replications {
+		trackedReplicationsMap[r.ID] = r.MaxQueueSizeBytes
+	}
+
+	// Queue manager completes startup tasks
+	if err := s.durableQueueManager.StartReplicationQueues(trackedReplicationsMap); err != nil {
+		return err
+	} else {
```

Comment:
Style nit: would typically omit the `else` wrapper here, and instead to something like you've done in `Close()` below:
```go
if err := stuff(); err != nil {
    return err
}
return nil
```